plugins/azure/fix_plugin_azure/resource/compute.py

```diff
@@ -3298,14 +3324,27 @@ def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
                     for ip_config in ip_configs:
                         if baps := ip_config.load_balancer_backend_address_pools:
                             for bap in baps:
-                                if bap_id := bap:
-                                    builder.add_edge(
-                                        self,
-                                        edge_type=EdgeType.default,
-                                        reverse=True,
-                                        clazz=AzureLoadBalancer,
-                                        id=bap_id,
-                                    )
+                                bap_id = "/".join(bap.split("/")[:-2])
+                                builder.add_edge(
+                                    self,
+                                    edge_type=EdgeType.default,
+                                    reverse=True,
+                                    clazz=AzureLoadBalancer,
+                                    id=bap_id,
+                                )
+                        if subnet_id := ip_config.subnet:
+                            builder.add_edge(
+                                self,
+                                edge_type=EdgeType.default,
+                                reverse=True,
+                                clazz=AzureSubnet,
+                                id=subnet_id,
+                            )
+        if vmss_instance_ids := self._vmss_vm_ids:
```

Comment:
See earlier commit: if you know the edge during collect, you do not need to wait for the connect phase.